Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -1509,19 +1509,25 @@ static bool parseCommandShortcut(const String& shortcut, OptionSet<ModifierFlags
         commandData.activationKey = emptyString();
         commandData.modifierFlags = { };
 
-        if (!hasActionCommand) {
-            if (supportsManifestVersion(3) && commandData.identifier == "_execute_action"_s)
-                hasActionCommand = true;
-            else if (!supportsManifestVersion(3) && (commandData.identifier == "_execute_browser_action"_s || commandData.identifier == "_execute_page_action"_s))
-                hasActionCommand = true;
-        }
+        bool isActionCommand = false;
+        if (supportsManifestVersion(3) && commandData.identifier == "_execute_action"_s)
+            isActionCommand = true;
+        else if (!supportsManifestVersion(3) && (commandData.identifier == "_execute_browser_action"_s || commandData.identifier == "_execute_page_action"_s))
+            isActionCommand = true;
+
+        if (isActionCommand && !hasActionCommand)
+            hasActionCommand = isActionCommand;
```

Comment:
Super nit: I think this is a bit more readable as `hasActionCommand = true`

Reply:
agreed!